Sources/FoundationExtensions/Collection/Dictionary+Extensions.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  Dictionary+Extensions.swift
+//  FoundationExtensions
+//
+//  Copyright © 2021 Lautsprecher Teufel GmbH. All rights reserved.
+//
+
+import Foundation
+
+extension Dictionary where Key: ExpressibleByStringLiteral {
+
+    /// Allows to use enum cases as a key for the subscript as long as the key of the Dictionary and the raw value of the enum
+    /// is `ExpressibleByStringLiteral`
+    ///
+    /// Example:
+    /// ```
+    /// enum Key: String {
+    ///     case foo
+    ///     case bar
+    /// }
+    ///
+    /// let someDict: [String: Any]
+    /// var foo = someDict[Key.foo]
+    public subscript<Index: RawRepresentable>(index: Index) -> Value? where Index.RawValue == String {
+        get {
+            return self[index.rawValue as! Key] // swiftlint:disable:this force_cast
```

Comment:
I guess we only have to cast to Key here because the generic is Key: ExpressibleByStringLiteral and not Key == String. I would change the generic constraint because if the key is ExpressibleByStringLiteral but not String, it will crash. For example, Substring.

Reply:
Key is now constrained to String. No force unwrapping needed.